### DISCUSSION TITLE
Mbed TLS in the n-able core?

### DISCUSSION THREAD

**Discussion by VAVorontsov** [h2zero/n-able-Arduino#57] [Q&A]
Hello,

Thank you for the great work on the new NRF core. 
I really like the idea behind it and use of the NimBLE library. 
There is great scope for BLE builds that are MCU agnostic, at least between NRF and ESP32.
Is there any possibility of adding the Mbed TLS library to the core?

Wireless comms inevitably benefit from encryption and it would be quite helpful to have the library integrated into the core in a similar way that it is integrated into the esp-idf on which the ESP32 Arduino core is based.

For example: Building [THIS CODE](https://github.com/Chreece/BTHomeV2-ESP32-example), that uses NimBLE library, works fully on ESP32, but I have to remove the encryption components for it to build using n-able core. I have tried to add the include and library files manually, but it got quite messy and I never got it to complete the build.

**Comment by h2zero** [comment]
Great suggestion, this should be doable. I have labeled this for addition.
